runtime/test/mock-slot-composer.js

```diff
@@ -51,19 +55,19 @@ class MockContext {
  * Usage example:
  *   mockSlotComposer
  *       .newExpectations()
- *           .expectRenderSlot('MyParticle1', 'mySlot1', ['template', 'model'])
- *           .thenSend('MyParticle1', 'mySlot1', '_onMyEvent', {key: 'value'})
- *       .newExpectations()
- *           .expectRenderSlot('MyParticle1', 'mySlot1', ['model'])
- *           .expectRenderSlot('MyParticle2', 'mySlot2', ['template', 'model'], (content) => !!content.myParam)
+ *           .expectRenderSlot('MyParticle1', 'mySlot1', ['template', 'model']);
+ *           .expectRenderSlot('MyParticle1', 'mySlot1', ['model'], 2)
+ *           .expectRenderSlotVerify('MyParticle2', 'mySlot2', (content) => !!content.myParam)
  *           .maybeRenderSlot('MyOptionalParticle', 'myOptionalSlot', ['template', 'model'])
+ *   mockSlotComposer.sendEvent('MyParticle1', 'mySlot1', '_onMyEvent', {key: 'value'});
  *   await mockSlotComposer.expectationsCompleted();
  */
 class MockSlotComposer extends SlotComposer {
-  constructor() {
+  constructor(options) {
```

Comment:
Document the possible options in `SlotComposer`? And here if they behave differently.

Reply:
done